cpp/oneapi/dal/algo/decision_forest/BUILD

```diff
@@ -16,30 +16,62 @@ dal_module(
     ]
```

Comment:
It's not PCA related file. Can this changes be separated into another PR?

Reply:
I've added support for DPC++ tests in Bazel to be able to run DPC++ PCA tests and had to fix existing rules/tests.